echo-core/src/main/java/com/netflix/spinnaker/echo/util/RetrofitUtils.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2025 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.echo.util;
+
+import okhttp3.HttpUrl;
+
+public class RetrofitUtils {
+
+  public static String getBaseUrl(String suppliedBaseUrl) {
```

Comment:
javadoc please.  Seems like we're gonna need this in kork, but this is fine at least for now.